src/billing-demo/src/config.rs

```diff
@@ -52,6 +62,14 @@ pub struct Args {
     /// Whether or not to run the billing-demo in a low memory mode
     #[structopt(long)]
     pub low_memory: bool,
+
+    /// A random seed for generating the records and prices
+    #[structopt(long)]
+    pub seed: Option<u64>,
+
+    /// A date to start generating records from. Default is a week before now
```

Comment:
you might want to define the expected format in the comment -- the comment string shows up as part of the `--help` output as well